src/MapAPI.php

```diff
@@ -71,7 +75,10 @@ class MapAPI extends Control
 	 */
 	private $direction = ['travelmode' => 'DRIVING'];
 	
-	
+	public function __construct(){
+		parent::__construct();
```

Comment:
Má nějakej význam ten prázdnej konstruktor? Kvůli tomu failujou testy a podle mě to nemůže mít žádnej vliv na funkčnost jestli to tam je nebo není...